static/css/dashboard.css

```
@@ -1,394 +1,495 @@
+.share-submit {
+    align-self: flex-start;
+}
+
+.visually-hidden {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
```

Stylelint (17.10.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)